docs/installation/install.rst

```diff
@@ -0,0 +1,65 @@
+============
+Installation
+============
+
+Requirements
+------------
+
+``mrinversion`` has the following strict requirements:
+
+- `Python <https://www.python.org>`_ 3.6 or later
```

Comment:
If I remember correctly, mrsimulator requires Python >= 3.7 now

Reply:
yeah. you are correct. We should also update the requirements file. Would you be up for updating this PR?

Reply:
Yes I can do that

Reply:
updated the python and numpy version